scripts/deploy_firebase.sh

```
@@ -0,0 +1,166 @@
+    done
+    echo
+    print_info "Run the iOS setup wizard: bash scripts/setup_ios_complete.sh"
+    exit 1
+fi
+
+# Load shared configuration
+print_info "Loading iOS shared configuration..."
+source secrets/shared_keys.env
+
+# Load Match password
+export MATCH_PASSWORD=$(cat secrets/.match_password)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)